tests/test_visitors/test_ast/test_classes/test_wrong_empty_lines_count.py

```diff
@@ -0,0 +1,108 @@
+"""Test method contain only allowed empty lines count."""
+import pytest
+
+from wemake_python_styleguide.violations.best_practices import (
+    WrongEmptyLinesCountViolation,
+)
+from wemake_python_styleguide.visitors.ast.function_empty_lines import (
+    WrongEmptyLinesCountVisitor,
+)
+
+class_with_wrong_method = """
+class WrongClass(object):
+
+    def wrong_method(self):
+        foo()
+
+        bar()
+
+        baz()
+
+        lighter()
+"""
+
+class_with_allow_method = """
```

Comment:
NIP: what do you think about using `valid` instead of `allow`?
`class_with_valid_method` sounds better

Reply:
Yes, I'm agree. My english level is not high, so I'm happy with any edits